backend/api/Database/Models/AnalysisResult.cs

```diff
@@ -0,0 +1,42 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Api.Database.Models
+{
+    public class AnalysisResult
+    {
+        [JsonPropertyName("inspectionId")]
```

Comment:
JsonProperty shouldn't be necessary in AnalysisResult

Reply:
It is necessary if we want to avoid making another DTO. Since it is already in the same format we want it in the frontend we can return it directly in SignalR. If we don't have this JsonPropertyName we can't be sure that it capitalizes words correctly.